### ISSUE TITLE
Fix incorrect textures

### ISSUE THREAD

**Issue by BobDotCom** [BobDotCom/B-1B#35]
Many texture files use incorrectly sized images. The dimensions should be powers of two, such as 512, 1024, 2048, etc.

Possibly related: #34

**Comment by RobAndrew96** [comment]
I did quite a lot of work back on this a while ago. I can take a look at textures again if need be?

**Comment by BobDotCom** [comment]
> I did quite a lot of work back on this a while ago. I can take a look at textures again if need be?

Yeah, we probably need to look over every single texture and make sure they're all sized to a factor of two. A script can probably be written to check that. I think @SammySkycrafts might have actually written one already?
